docs/installation.rst

```diff
@@ -1,8 +1,14 @@
 User Setup
 ==========
 
-Install pulp-python
--------------------
+Ansible Installer (Recommended)
+-------------------------------
```

Comment:
This could be a separate commit if anyone cares. Just fixed it when I noticed it was missing.